pkg/images/defaults/flavor.go

```diff
@@ -190,6 +195,39 @@ func RHACSReleaseImageFlavor() ImageFlavor {
 	}
 }
 
+// OpenSourceReleaseImageFlavor returns image values for `opensource` flavor.
+func OpenSourceReleaseImageFlavor() ImageFlavor {
+	v := version.GetAllVersionsDevelopment()
+	return ImageFlavor{
+		MainRegistry:       "quay.io/stackrox-io",
+		MainImageName:      "main",
+		MainImageTag:       v.MainVersion,
+		CentralDBImageTag:  v.MainVersion,
+		CentralDBImageName: "central-db",
+
+		CollectorRegistry:      "quay.io/stackrox-io",
+		CollectorImageName:     "collector",
+		CollectorImageTag:      v.CollectorVersion + "-latest",
+		CollectorSlimImageName: "collector",
+		CollectorSlimImageTag:  v.CollectorVersion + "-slim",
```

Comment:
Seems like the collector images are incorrect. The `quay.io` images follow the same pattern as `stackrox.io`: the `-slim` suffix belongs to the name not the tag (see [here](https://quay.io/organization/stackrox-io)). Please update to:
```go
CollectorImageName:     "collector",
CollectorImageTag:      v.CollectorVersion,
CollectorSlimImageName: "collector-slim",
CollectorSlimImageTag:  v.CollectorVersion,
```

Reply:
Oh, that's a great catch, thanks!
Let me update this and re-test the helm deployment locally to make sure everything works as intended!